docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -124,7 +124,14 @@ respective latest released distribution tarballs from the [ZNMSTR] account on
 ## 8. Produce distribution zip file (Zonemaster-GUI only)
 
 The requirements are nodejs and npm. There are available from the [official website]( https://nodejs.org/en/).
-Minimal version of Nodejs is 6.0 but install the last LTS version available.
+Minimal version of Nodejs is 10.0 but install the last LTS version available.
+
```

Comment:
Please add information on what OS and version that this will work on, e.g. Ubuntu 18.04 or CentOS 8 or whatever your suggestion is. It would be preferred if the suggested OS and version is one that we already have for Zonemaster but it is not a requirement.

Reply:
If I start with a clean Ubuntu 18.04, do I have to install anything besides `git` and `curl` before running the `nvm` commands?